apps/meteor/app/livechat/server/lib/Visitors.ts

```diff
@@ -0,0 +1,97 @@
+import { UserStatus, type ILivechatVisitor } from '@rocket.chat/core-typings';
+import { LivechatDepartment, LivechatVisitors } from '@rocket.chat/models';
+
+import { settings } from '../../../settings/server';
+import { validateEmail } from './Helper';
+import { Livechat, type RegisterGuestType } from './LivechatTyped';
```

Comment:
This file is also imported in `livechattyped`, so a circular ref.

can we break it? so this file doesnt import anything from there.